src/en/wip-storage.md

```diff
@@ -185,33 +209,16 @@ IMPLEMENTED FEATURES
   * list storage instances/attachments
   * list volumes/attachments
   * list and create storage pools
-- probably other things which elude me
-
-PROVIDER SUPPORT
--------------------------------
-
-All environment providers support the following storage providers:
- - loop: block-kind, creates a file in the agent data-dir and attaches a loop device to it. See the caveats section below for a comment on using the loop storage provider with local/LXC.
- - rootfs: filesystem-kind, creates a sub-directory in the agent's data-dir for the unit/charm to use
-
-We have implemented support for creating volumes in the ec2 provider, via the "ebs" storage provider. By default, the ebs provider will create cheap and nasty magnetic volumes. There is also an "ebs-ssd" storage pool provided OOTB that will create SSD (gp2) volumes. Finally, you can create your own pools if you like; the parameters for ebs are:
-  - volume-type: may be "magnetic", "ssd", or "provisioned-iops"
-  - iops: number of provisioned IOPS (requires volume-type=provisioned-iops)
-
-Some storage providers also support a "persistent=<bool>" pool attribute. By using this, Juju will not tie the lifetime of storage entities (volumes, filesystems) to the lifetime of the machines that they are attached to. In EC2/EBS terms, a persistent volume is one without any attachments having the DeleteOnTermination flag set. Juju will not allow you to cleanly destroy an environment with persistent volumes; you may use "--force" to subvert this as usual, but please be aware that this will leak the resource.
 
 UNIMPLEMENTED/CAVEATS
```

Comment:
already did:
"Charm upgrade does not currently check for incompatible changes to storage requirements in deployed charms."